app/lib/submission_builder/ty2022/states/az/dependent_relationship_table.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+module SubmissionBuilder
+  module Ty2022
+    module States
+      module Az
+        module DependentRelationshipTable
+          # "direct file relationship" => "AZ XML relationship"
+          DF_TO_AZ_XML_RELATIONSHIPS = {
+            "DAUGHTER" => "CHILD",
+            "STEPCHILD" => "CHILD",
+            "FOSTER CHILD" => "FOSTERCHILD",
+            "GRANDCHILD" => "GRANDCHILD",
+            "SISTER" => "SIBLING",
+            "NEPHEW" => "NIECENEPHEW",
+            "HALF" => "SISTER	HALFSIBLING",
```

Comment:
Typo here? Perhaps it should be:
```suggestion
            "HALF SISTER" => "HALFSIBLING",
```

Reply:
Great catch! Thank you!